### ISSUE TITLE
When do you support return value types

### ISSUE THREAD

**Issue by AI-pig** [Manweill/swagger-axios-codegen#26]

**Comment by Manweill** [comment]
what is the problem

**Comment by perminov-aleksandr** [comment]
Hello, im not a author of issue, but i struggle with the same problem.
Generated class does not contain types for result when it is specified.
For example, i have a path in specification file that returns array of AppRole type:
```json
"paths": {
    "/api/Roles/Get": {
      "get": {
        "responses": {
          "200": {
            "description": "Success",
            "schema": {
              "uniqueItems": false,
              "type": "array",
              "items": {
                "$ref": "#/definitions/AppRole"
              }
            }
          }
        }
      }
    }
},
"definitions": {
    "AppRole": {
      "type": "object",
      "properties": {
        "id": {
          "type": "string"
        },
        "name": {
          "type": "string"
        }
      }
    }
}
```

but result is generated service method without type, just Promise of **any**

```typescript
export class RolesService {
static get(options: IRequestOptions = {}): Promise<any> {
    return new Promise((resolve, reject) => {
    //...
  }
}
```

**Comment by Manweill** [comment]
@perminov-aleksandr 
ok, I see;

**Comment by Manweill** [comment]
@perminov-aleksandr @AI-pig fix in v0.5.7
